tests/server_test_cases/person_note_tests.py

```diff
@@ -497,34 +497,56 @@ def assert_params(url=None):
             full_name='_test_given_name _test_family_name',
             details={'Author\'s name:': '_test_author_name'})
 
-        # Now the search should yield a result.
+    @parameterized.expand([(True,), (False,)])
+    def test_search_with_result(self, use_full_text_search):
+        config.set(enable_full_text_search = use_full_text_search)
+        def assert_params(url=None):
+            self.assert_params_conform(
+                url or self.s.url, {'role': 'seek'}, {'ui': 'small'})
+        self.go('/haiti/create?query=&role=seek&given_name=&family_name=')
+        self.submit_minimal_create_form(self.s.doc.cssselect_one('form'))
+        search_page = self.go('/haiti/query?role=seek')
+        search_form = search_page.cssselect_one('form')
         self.s.submit(search_form, query_name='_test_given_name')
         assert_params()
         self.verify_results_page(1, all_have=(['_test_given_name']),
                                  some_have=(['_test_given_name']),
                                  status=(['Unspecified']))
         self.verify_click_search_result(0, assert_params)
-        # set the person entry_date to something in order to make sure adding
-        # note doesn't update
+
+    def test_note_doesnt_update_entry_date(self):
+        self.go('/haiti/create?query=&role=seek&given_name=&family_name=')
+        self.submit_minimal_create_form(self.s.doc.cssselect_one('form'))
         person = Person.all().filter('given_name =', '_test_given_name').get()
-        person.entry_date = datetime.datetime(2006, 6, 6, 6, 6, 6)
-        db.put(person)
+        entry_date = person.entry_date
         self.verify_details_page(num_notes=0)
+        self.advance_utcnow(days=5)
+        self.verify_update_notes(
+            False, '_test A note body', '_test A note author', None)
```

Comment:
might be worth explicitly using keywords for these arguments, since it is otherwise difficult to tell what False and None are doing. (this applies a few other places in this file)

Reply:
Done.